src/clr-angular/forms/styles/_range.clarity.scss

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.range') {
+  .clr-range-wrapper {
+    position: relative;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-end;
+    height: $clr-forms-range-thumb-height;
+    white-space: nowrap;
+
+    .fill-input {
+      position: absolute;
+      left: 0;
+      display: inline-block;
+      height: $clr-forms-range-track-height;
+      background-color: $clr-forms-range-progress-fill-color;
+      pointer-events: none;
+      cursor: pointer;
+      z-index: 10;
+    }
+  }
+
+  .clr-range {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    height: $clr-forms-range-thumb-height;
+  }
+
+  input[type='range'] {
+    @include range-style($clr-forms-range-track-height, $clr-forms-range-track-color);
+    padding: 0;
+
+    &::-webkit-slider-runnable-track {
+      @include range-track-style($clr-forms-range-track-height, $clr-forms-range-track-color);
+    }
+
+    &::-webkit-slider-thumb {
+      -webkit-appearance: none;
+      @include range-thumb-style(
+        $clr-forms-range-thumb-margin,
+        $clr-forms-range-thumb-height,
+        $clr-forms-range-progress-fill-color
+      );
+    }
+
+    &::-moz-range-track {
+      @include range-track-style($clr-forms-range-track-height, $clr-forms-range-track-color);
+    }
+
+    &::-moz-range-thumb {
+      @include range-thumb-style(
+        $clr-forms-range-thumb-margin,
+        $clr-forms-range-thumb-height,
+        $clr-forms-range-progress-fill-color
+      );
+      border: 0;
+    }
+  }
+
+  @include fixForMsEdge() {
+    .clr-range-wrapper {
+      .fill-input {
+        display: none;
+      }
+
+      &.progress-fill {
+        input[type='range']::-ms-fill-lower {
+          background: $clr-forms-range-progress-fill-color;
```

Comment:
We will need to use custom properties here. Check out how @include css-var() is used in other SASS files. background-color, background, and border-radius should all be allowed to use custom properties.